webrtc-stats.html

```diff
@@ -3114,6 +3170,8 @@ <h3>
              RTCDtlsRole           dtlsRole;
              DOMString             srtpCipher;
              unsigned long         selectedCandidatePairChanges;
+             unsigned long         ccfbMessagesSent;
+	     unsigned long         ccfbMessagesReceived;
```

Comment:
Bike-shedding: unsure about "messages" (RFC 8888 calls them packets)

The first one counts feedback sent from the receiver to the sender, is that right?

Might it be clearer to call them something like:
```suggestion
             unsigned long         rtcpCcFeedbackSent;
             unsigned long         rtcpCcFeedbackReceived;
```
to clarify RTCP goes in the other direction (and leaning on the word "feedback")?

Reply:
The RFC uses "messages" in section 3.1
The RFC uses "ccfb" as the identifier in SDP, I don't want to deviate from that.